ngraph_onnx/onnx_importer/utils/reshape.py

```diff
@@ -92,3 +113,41 @@ def flatten_innermost_empty_dims(node):  # type: (NgraphNode) -> NgraphNode
         return ng.reshape(node, input_order, output_shape)
     else:
         return node
+
+
+def get_bound(requested_bound, max_bound_val):  # type: (int, int) -> int
+    """Return valid bound value withing range [0,max_bound_val].
```

Comment:
What is a bound value? Please use a different term in docstring to explain what you mean.
Alternativel add another sentence to explain what bound means in this context.
What are we using this for?